### ISSUE TITLE
Narrator does not announce the label and message below the edit box of 'Body, edit field'.

### ISSUE THREAD

**Issue by msftedad** [tinymce/tinymce#8505]
**What is the current behavior? Describe the bug**
Upon navigating to 'Body, edit field', narrator announces as 'editing, new line'. It does not announce the label for the edit field. Also, it does not announce the below message 'Recommended font Segeo UI with 16px'.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
Step 1: Please open https://codepen.io/keyapanja/pen/NWBwzYw in Edge browser and turn ON Narrator.
Step 2: The default focus would be on HTML body
Step 3: Press ESC, the focus would be on 'Open HTML settings'
Step 4: Press shortcut key 'B' 6 times to navigate to 'Console ' button.
Step 5: Press Shift+Tab to navigate to 'Body' edit field and observe the announcement.

**What is the expected behavior?**
Upon navigating to 'Body, edit field', narrator should announce the label for the edit field and also the below message as 'Body, editing, new line, Recommended font Segeo UI with 16px'.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
OS Version: Windows 11 Dev 22H2(OS Build 22621.1265 )
Edge Dev Browser Version: 110.0.1587.57 (Official build) (64-bit) 
Tiny MCE version V6.0
![body edit field](https://user-images.githubusercontent.com/93735775/222404212-ef8d5b16-849c-41cf-be30-93b75a9c215e.jpg)
![dev tool-body field](https://user-images.githubusercontent.com/93735775/222404223-73087eb9-3144-4bb1-a74c-c86c2e0682a5.jpg)

**Comment by msftedad** [comment]
Hi Team,

Any update on this issue, waiting for your response.

Thanks.

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team,

Please respond to the comments.

Thanks

**Comment by tiny-ci-servers** [comment]
This issue is stale because it has been open 365 days with no activity. Please comment if you wish to keep this issue open or it will be closed in 30 days.
